src/legacy/core_plugins/console/server/proxy_route.js

```diff
@@ -68,10 +70,6 @@ export const createProxyRoute = ({
   method: 'POST',
   config: {
     tags: ['access:console'],
-    payload: {
```

Comment:
If we leave this configuration in place, GET requests result in a circular JSON error:

```
Debug: internal, implementation, error
    TypeError: Converting circular structure to JSON
    at JSON.stringify (<anonymous>)
    at stringify (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/src/legacy/core_plugins/console/server/proxy_route.js:131:32)
    at new Promise (<anonymous>)
    at handler (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/src/legacy/core_plugins/console/server/proxy_route.js:114:20)
    at module.exports.internals.Manager.execute (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/node_modules/hapi/lib/toolkit.js:35:106)
    at Object.internals.handler (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/node_modules/hapi/lib/handler.js:50:48)
    at exports.execute (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/node_modules/hapi/lib/handler.js:35:36)
    at process._tickCallback (internal/process/next_tick.js:68:7)
server   error  [15:30:59.723]  TypeError: Converting circular structure to JSON
    at JSON.stringify (<anonymous>)
    at stringify (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/src/legacy/core_plugins/console/server/proxy_route.js:131:32)
    at new Promise (<anonymous>)
    at handler (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/src/legacy/core_plugins/console/server/proxy_route.js:114:20)
    at module.exports.internals.Manager.execute (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/node_modules/hapi/lib/toolkit.js:35:106)
    at Object.internals.handler (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/node_modules/hapi/lib/handler.js:50:48)
    at exports.execute (/Users/cjcenizal/Documents/GitHub/Elastic/kibana/node_modules/hapi/lib/handler.js:35:36)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```